src/python/TaskWorker/Actions/PreJob.py

```diff
@@ -553,7 +546,7 @@ def execute(self, *args):
         ## Note the cooloff time is based on the DAGMan retry number (i.e. the number of
         ## times the full cycle pre-job + job + post-job finished). This way, we don't
         ## punish users for condor re-starts.
-        sleep_time = int(self.dag_retry)*60
+        sleep_time = 60 #int(self.dag_retry)*60
```

Comment:
we will need to revisit this sleep_time. As it is (3 min max) makes little sense, too short to have visible effects. Yet is there since original version by Brian B. who surely knew what he what doing. But ten years later we also know and are allowed to revisit.